.erda/migrations/pipeline/20220525-pipeline-cron-add-cluster-name.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `pipeline_crons` ADD COLUMN `cluster_name` varchar(64) DEFAULT NULL COMMENT '集群名称';
+ALTER TABLE `pipeline_crons` ADD INDEX `idx_cluster_name` (`cluster_name`);
```

Comment:
I think this index is not valuable.
Test combined index.

Reply:
adusted, combine 'cluster_name','pipeline_yml_name','pipeline_source'